src/metadata/routes/tags.clj

```diff
@@ -10,6 +10,33 @@
   (context "/filesystem/data" []
     :tags ["tags"]
 
+    (GET "/tags" []
+      :query [{:keys [user]} StandardUserQueryParams]
+      :responses {200      {:schema      AttachedTagsListing
+                            :description "Attached tags are listied in the response"}
+                  500      {:schema      ErrorResponseUnchecked
+                            :description "Unchecked errors"}
+                  :default {:schema      ErrorResponse
+                            :description "All other errors"}}
+      :summary "List All Attached Tags for a User"
+      :description
+      "This endpoint lists all tags that have been attached to a file or folder by a user."
+      (ok (tags/list-all-attached-tags user)))
+
+    (DELETE "/tags" []
+      :query [{:keys [user]} StandardUserQueryParams]
+      :responses {200      {:schema      (describe String "Empty response")
```

Comment:
Another case where I couldn't find a better way to get compojure-api to handle this.